openid-4-verifiable-presentations-1_0.md

```diff
@@ -73,6 +73,9 @@ This specification also defines the following terms. In the case where a term ha
 Credential:
 :  A set of one or more claims about a subject made by a Credential Issuer. Note that the definition of the term "Credential" in this specification is different from that in [@!OpenID.Core].
 
+Digital Credentials API:
+:  The Digital Credentials API (DC API) refers to the W3C Digital Credentials API [@!W3C.Digital_Credentials_API] on the Web Platform and its equivalent native APIs on App Platforms (such as [Credential Manager on Android](https://developer.android.com/jetpack/androidx/releases/credentials)).
```

Comment:
is it ok to point to a specific implementation in the definition..? re `Credential Manager on Android`

Reply:
I think it's probably okay, it does seem useful to point developers in the right directions. Ideally it would be good to include more than one (Apple?) but is Google the only one that has written anything public right now?

We can always add more references for other OSes later, potentially even in errata after we've gone final.